packages/wallet-sdk/src/sign/walletlink/relay/ui/components/RedirectDialog/RedirectDialog.tsx

```diff
@@ -1,10 +1,10 @@
-import clsx from 'clsx';
-import { FunctionComponent, h, render } from 'preact';
```

Comment:
I believe removing the h pragma will break the preact UI

Reply:
Unsure how this got removed, adding back in.

Reply:
I think prettier might auto remove it. I have a task to fix up our eslint/prettier. Looks like you suggested biome. Will take a look into that.